skbuild/resources/cmake/UseCython.cmake

```diff
@@ -286,8 +298,12 @@ function(add_cython_target _name)
       list(REMOVE_DUPLICATES include_dependencies)
 
       # Add modules to the files to check, if appropriate.
-      foreach(module ${module_dependencies})
+      foreach(module IN LISTS module_dependencies)
         unset(pxd_location CACHE)
+        string(REPLACE "." "/" module ${module})
+        # TODO: Handle relative import paths, as well as establishing
+        #       The project root directory
+
```

Comment:
I'm not exactly sure how to handle this. The module appears designed to find `x.y.z.pxd` files, but in reality modern Cython uses `x/y/z.pxd` files. What if we need to find the pxd from `cimport x.y.z`? Are we to assume that it could be in `${CMAKE_SOURCE_DIR}/x/y/z.pxd`? What justifies that assumption, especially with gh-296?

Thoughts @jcfr?